releases/release-1.31/release-notes/release-notes-draft.json

```diff
@@ -229,8 +220,29 @@
       "instrumentation"
     ],
     "duplicate": true,
-    "duplicate_kind": true,
-    "is_mapped": true
+    "duplicate_kind": true
+  },
+  "122025": {
```

Comment:
Example of new release note that linked to the map.

Reply:
thanks !